docsets/FileMaker/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "FileMaker",
-    "version": "14.0.1",
+    "version": "15.0.0",
     "archive": "FileMaker.tgz",
```

Comment:
You need to add the docset archive at this path too, not just in the `versions` folder.